django/db/models/aggregates.py

```diff
@@ -20,12 +21,14 @@ class Aggregate(Func):
     filter_template = '%s FILTER (WHERE %%(filter)s)'
     window_compatible = True
     allow_distinct = False
+    empty_aggregate_value = None
 
-    def __init__(self, *expressions, distinct=False, filter=None, **extra):
+    def __init__(self, *expressions, distinct=False, filter=None, default=None, **extra):
         if distinct and not self.allow_distinct:
             raise TypeError("%s does not allow distinct." % self.__class__.__name__)
         self.distinct = distinct
         self.filter = filter
+        self.default = default
```

Comment:
What about

```suggestion
        if default is not None and self.empty_aggregate_value is not None:
            raise ValueError(f'{self.__class__.__name__} does not support the default argument.')
        self.default = default
```

Reply:
I went with the following underneath the equivalent check for `distinct` for consistency:
```python
        if default is not None and self.empty_aggregate_value is not None:
            raise TypeError(f'{self.__class__.__name__} does not allow default.')
```